docs/src/getting-started/installation.md

```diff
@@ -46,6 +52,14 @@ asdf install starknet-foundry latest
 
 See [asdf guide](https://asdf-vm.com/guide/getting-started.html) for more details.
 
+### Universal-Sierra-Compiler update
+
+If you would like to bump the USC manually (e.g. when the new Sierra version is released) you can do it by running:
+
+```shell
+curl -L https://raw.githubusercontent.com/software-mansion/universal-sierra-compiler/master/scripts/install.sh | sh
```

Comment:
Will the script make sure that the latest installed USC will be used? cc @THenry14

Reply:
It installs (v1.0.0) for me.

Reply:
Yea, I think the script is meant to install the latest stable version, which is 1.0.0 rn xD We need to release 2.0.0

My question was if a user has 1.0.0 installed. Let's say 2.0.0 is released so this script installs 2.0.0.
Will the version 2.0.0 be selected when someone runs `universal-sierra-compiler` after installing the newest version?

Reply:
Yup, the newest version will be used